include/zephyr/net/wifi_mgmt.h

```diff
@@ -274,6 +289,14 @@ enum wifi_twt_sleep_state {
 	WIFI_TWT_STATE_AWAKE = 1,
 };
 
+struct wifi_listen_interval_params {
+	unsigned short listen_interval;
```

Comment:
As discussed earlier, having a OP for each param is an overkill, can you please group these parameters and simplify the ops?